x/data/server/testsuite/suite.go

```diff
@@ -35,6 +38,7 @@ func (s *IntegrationTestSuite) SetupSuite() {
 
 	s.fixture = s.fixtureFactory.Setup()
 	s.ctx = s.fixture.Context()
+	s.sdkCtx = s.ctx.(types.Context).WithBlockTime(time.Now())
```

Comment:
we need to update `s.ctx` too as in line 93, not sure `time.Now` is smart for tests

Reply:
Updated in https://github.com/regen-network/regen-ledger/pull/971/commits/c14f22b14f01fddfd0a8527daf856e9574ed6b8b